src/components/Interfaces.tsx

```diff
@@ -1,28 +1,9 @@
-// import { To } from "react-router-dom";
-// export interface trainingItemProps {
-//     id: To;
-//     trainingName: string;
-//     category: string;
-//     date: string;
-// }
-
-export interface FormDataInterface {
-    firstName: string;
-    lastName: string;
-    email: string;
-    password: string;
-    repasswd: string;
-    gender: string;
-    agree: boolean;
-    formErrors: {
-        firstName: string;
-        lastName: string;
-        email: string;
-        password: string;
-        repasswd: string;
-    };
-    emailValid: boolean;
-    passwordValid: boolean;
-    repasswdValid: boolean;
-    formValid: boolean;
-}
\ No newline at end of file
+export type FormDataType = {
+  firstName: string;
+  lastName: string;
+  email: string;
+  password: string;
+  repasswd: string;
+  gender: 'female' | 'male';
+  termsAndConditions: boolean;
+};
```

Comment:
Can we use User from models instead of this? Should I add gender field and field for acceptation terms and conditions?

Reply:
That's good question..
I think it's better to track these fields and add them to database

Reply:
Then I'll change type to user from models and everything will work completely fine